tests/test_integration_queue.py

```diff
@@ -0,0 +1,108 @@
+"""
+End-to-end integration tests with OpenReview server and Celery task queue.
+"""
+
+import json
+import time
+
+import openreview
+
+from conftest import clean_start_conference, wait_for_status
+
+
+def test_integration_basic(openreview_context, celery_app, celery_worker):
+    """
+    Basic integration test. Makes use of the OpenReview Builder
+    """
+    openreview_client = openreview_context["openreview_client"]
+    test_client = openreview_context["test_client"]
+
+    num_reviewers = 20
+    num_papers = 20
+    reviews_per_paper = 3
+    max_papers = 5
+    min_papers = 1
+    alternates = 0
+
+    notes_ids = []
+    for i in range(10):
+        conference_id = "AKBC.ws/{year}/Conference".format(year=3000 + i)
+        conference = clean_start_conference(
```

Comment:
you don't need to clean and start the conference for every run. You can use the same conference to run different matchings.

Reply:
I was getting No Solution with using the same conference.